src/pms.cpp

```diff
@@ -744,7 +744,9 @@ int			Pms::init()
 	fieldtypes->add("trackshort", _("No"), FIELD_TRACKSHORT, 3, sort_compare_track);
 	fieldtypes->add("length", _("Length"), FIELD_TIME, 7, sort_compare_length);
 	fieldtypes->add("date", _("Date"), FIELD_DATE, 11, sort_compare_date);
+	fieldtypes->add("originaldate", _("Original date"), FIELD_ORIGINALDATE, 0, sort_compare_originaldate);
 	fieldtypes->add("year", _("Year"), FIELD_YEAR, 5, sort_compare_year);
+	fieldtypes->add("originalyear", _("Orig"), FIELD_ORIGINALYEAR, 5, sort_compare_originalyear);
```

Comment:
But then we might have two columns both titled "Year". I prefer them to be different, but will concede to your decision.